### PR TITLE
[WIP] Removes shoulda-context from applications test

### PR DESCRIPTION
**Pull request by josemigallas**
Wait for https://github.com/3scale/porta/pull/2748

### CODE REVIEW

**Review by akostadinov — APPROVED** [review]
Looks good. Honestly I'm not sure I could track all the refactored tests ¯\_(ヅ)_/¯

File: test/integration/user-management-api/applications_test.rb [hunk 1]
```diff
@@ -34,255 +34,280 @@ def setup
...
 
-      should 'pagination per_page has a maximum allowed' do
-        get admin_api_applications_path(format: :xml), params: params.merge({ per_page: max_per_page + 1 })
+      get admin_api_applications_path(format: :xml), params: params
+      assert_response :success
 
-        assert_response :success
-        assert_pagination @response.body, "applications", per_page: max_per_page
-      end
+      assert_applications_count @response.body, total_entries
```
Comment (akostadinov) [inline test/integration/user-management-api/applications_test.rb:130]: Actually where is this defined? Tests fail with no such method.

Comment (josemigallas) [reply test/integration/user-management-api/applications_test.rb:130]: Yeah I put it into a stash and forgot about it 😅

**Comment by josemigallas** [comment]
> Looks good. Honestly I'm not sure I could track all the refactored tests ¯_(ヅ)_/¯

It's a leap of faith haha

**Comment by josemigallas** [comment]
No need for this anymore, context removed in https://github.com/3scale/porta/pull/2748/files

**Pull request closed without merging**